go/cmd/vtctldclient/command/vreplication/materialize/create.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package materialize
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"vitess.io/vitess/go/cmd/vtctldclient/cli"
+	"vitess.io/vitess/go/cmd/vtctldclient/command/vreplication/common"
+	"vitess.io/vitess/go/vt/sqlparser"
+	"vitess.io/vitess/go/vt/topo/topoproto"
+
+	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
+)
+
+var (
+	createOptions = struct {
+		SourceKeyspace string
+		TableSettings  tableSettings
+	}{}
+
+	// create makes a MaterializeCreate gRPC call to a vtctld.
+	create = &cobra.Command{
+		Use:                   "create",
+		Short:                 "Create and run a Materialize VReplication workflow.",
```

Comment:
TIOLI: might be worth expanding on the syntax for `--table-settings` in a `Long` text (essentially to replace/generate [this](https://vitess.io/docs/19.0/reference/vreplication/materialize/#json-spec-details)

Reply:
Good idea! I'll do that.